MAUI/DataForm/floating-label-layout.md

```diff
@@ -0,0 +1,777 @@
+                        DataObject="{Binding ContactInfo}"
+                        GenerateDataFormItem="OnGenerateDataFormItem">
+    <dataForm:SfDataForm.TextInputLayoutSettings>
+        <dataForm:TextInputLayoutSettings ShowHelperText="True">
+            <dataForm:DataFormTextItem.LabelTextStyle>
+                <dataForm:DataFormTextStyle TextColor="CadetBlue" FontAttributes="None"/>
+            </dataForm:DataFormTextItem.LabelTextStyle>
+        </dataForm:TextInputLayoutSettings>
+    </dataForm:SfDataForm.TextInputLayoutSettings>
+</dataForm:SfDataForm>
+</ContentPage>
+{% endhighlight %}
+{% highlight C# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+{
+    if (e.DataFormItem != null && e.DataFormItem.FieldName == "Name")
+    {
+        e.DataFormItem.LayoutType = DataFormLayoutType.TextInputLayout;
+        e.DataFormItem.LabelTextStyle = new DataFormTextStyle { FontAttributes = FontAttributes.Bold, TextColor = Colors.BlueViolet };
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Helper label style
+You can customize the text of helper label by setting the TextColor, FontFamily, FontSize, and FontAttributes properties of `HelperTextStyle` in DataFormItem.
+
+{% tabs %}
+{% highlight XAML %}
+<ContentPage 
+...
+xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm">
+<dataForm:SfDataForm x:Name="dataForm" 
+                        LayoutType="TextInputLayout"
+                        DataObject="{Binding ContactInfo}"
+                        GenerateDataFormItem="OnGenerateDataFormItem">
+    <dataForm:SfDataForm.TextInputLayoutSettings>
+        <dataForm:TextInputLayoutSettings ShowHelperText="True">
+            <dataForm:TextInputLayoutSettings.HelperTextStyle>
+                <dataForm:DataFormTextStyle FontAttributes="Italic" TextColor="Violet" />
+            </dataForm:TextInputLayoutSettings.HelperTextStyle>
+        </dataForm:TextInputLayoutSettings>
+    </dataForm:SfDataForm.TextInputLayoutSettings>
+</dataForm:SfDataForm>
+</ContentPage>
+{% endhighlight %}
+{% highlight C# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+{
+    if (e.DataFormItem != null && e.DataFormItem.FieldName == "Name")
+    {
+        e.DataFormItem.LayoutType = DataFormLayoutType.TextInputLayout;
+        e.DataFormItem.TextInputLayoutSettings = new TextInputLayoutSettings
+        {
+            HelperTextStyle = new DataFormTextStyle { TextColor = Colors.Blue, FontAttributes = FontAttributes.Bold },
+        };
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Error message label style
+You can customize the text of error message label by setting the TextColor, FontFamily, FontSize, and FontAttributes properties of `ErrorLabelTextStyle` in DataFormItem.
```

Comment:
Customize the text of the error message label by setting the TextColor, FontFamily, FontSize, and FontAttributes properties of the `ErrorLabelTextStyle` in the DataFormItem.

Reply:
Content changed